examples/index.html

```diff
@@ -208,9 +208,9 @@ <h2>Extensibility</h2>
     </div>
 </section>
 
-<footer class="navbar bg-dark p-2">
+<!--<footer class="navbar bg-dark p-2">-->
```

Comment:
@tbouffard do we need that?

Reply:
No, there is no reason to keep the comment.
I am removing it

Reply:
done with 9ce9557